config/s2i/jenkins/master/configuration/hudson.plugins.ircbot.IrcPublisher.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<hudson.plugins.ircbot.IrcPublisher_-DescriptorImpl plugin="ircbot@2.30">
+    <enabled>true</enabled>
+    <hostname>irc.freenode.net</hostname>
+    <port>6667</port>
+    <ssl>false</ssl>
+    <disallowPrivateChat>false</disallowPrivateChat>
+    <login>contra-bot</login>
+    <sslTrustAllCertificates>false</sslTrustAllCertificates>
+    <password>cjNkaEB0</password>
```

Comment:
not sure about this...seems like bad idea having this in SCM since the account can merge our builds.

Reply:
@scoheb This isn't the real password.  I am open to other ideas or do we have to wait until we have secrets in Openshift to pass to Jenkins?